### PR TITLE
Backports master

### PR DESCRIPTION
**Pull request by mkralik3**
* added timeout after deleting extension
* fixed reset DV
* added all failed build to the log

//skip-ci
Backport for 
https://github.com/syndesisio/syndesis-qe/pull/1529
https://github.com/syndesisio/syndesis-qe/pull/1530
https://github.com/syndesisio/syndesis-qe/pull/1537

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/soak-deployments/maxIntegrations.feature [hunk 1]
```diff
@@ -162,6 +153,16 @@ Feature: Test maximum integration
...
+    
+    And wait until integration "Integration1" gets into "Running" state
+    And wait until integration "Integration2" gets into "Running" state
+    And wait until integration "Integration3" gets into "Running" state
+    And wait until integration "Integration4" gets into "Running" state
+    And wait until integration "Integration5" gets into "Running" state
+    And wait until integration "Integration6" gets into "Running" state
+    And wait until integration "Integration7" gets into "Running" state
+    And wait until integration "Integration8" gets into "Running" state
+    And wait until integration "Integration9" gets into "Running" state
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/soak-deployments/maxIntegrations.feature:165]: Great! now the test may be able to finish under a day :D

**Pull request merged**